locales/fr_CA/LC_MESSAGES/duckduckgo.po

```diff
@@ -4218,7 +4218,7 @@ msgstr "Météo"
 
 msgctxt "feedback form"
 msgid "Web Links"
-msgstr "LIens internet"
+msgstr "LIens Internet"
```

Comment:
Translation has changed - we should leave as-is (lowercase).

BUT... this should be changed to "Liens" (no uppercase "i").